packages/instantsearch.js/src/components/InfiniteHits/InfiniteHits.tsx

```diff
@@ -1,6 +1,6 @@
 /** @jsx h */
 
-import { cx } from '@algolia/ui-components-shared';
+import { cx } from 'instantsearch-ui-components';
```

Comment:
we should ensure when we do a release that React InstantSearch and Vue InstantSearch migrate to this as well soon, so there's no duplicate implementations of cx (although it doesn't matter much as the function is tiny)

Reply:
Yes, I'll replace it progressively in other PRs.